autonomy/deploy/generators/kubernetes/templates.py

```diff
@@ -251,8 +251,6 @@
             value: "1"
           - name: LOG_FILE
             value: "/logs/aea_{validator_ix}.txt"
-          - name: PYTHONHASHSEED
```

Comment:
see above

Reply:
Reverted change in 25abec60251a55b9d5e1a763c5b9be6290e8eef8.